vivarium/core/store.py

```diff
@@ -1021,13 +1024,11 @@ def emit_data(self):
         if self.emit:
             if self.serializer:
                 if isinstance(self.value, list) and self.units:
-                    return self.serializer.serialize(
-                        [v.to(self.units) for v in self.value])
+                    return [self.serializer.serialize(v.to(self.units))
+                            for v in self.value]
                 if self.units:
                     return self.serializer.serialize(
                         self.value.to(self.units))
-                if isinstance(self.value, list):
```

Comment:
Matt added this in a PR some time back to fix vivarium save states. The save and load state test in `vivarium-ecoli` ran just fine now with these lines removed.